packages/terra-pills/src/private/_RollupPill.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {
+  useRef,
+} from 'react';
+import {
+  KEY_SPACE, KEY_RETURN,
```

Comment:
Same question here about testing keyboard commands in FF browser.

Reply:
Tested this on FF. Works fine.